vmdb/app/models/vm_or_template.rb

```diff
@@ -1102,7 +1102,7 @@ def miq_proxies
 
   # Cache the servers because the JobProxyDispatch calls this for each Vm scan job in a loop
   cache_with_timeout(:miq_servers_for_scan, 30.seconds) do
-    MiqServer.find(:all, :conditions => {:status => "started"}, :include => [:zone, :server_roles])
+    MiqServer.where(:status => "started").includes([:zone, :server_roles]).to_a
```

Comment:
This one was kept in because it wasn't clear we would want to cache a AR::Relation.